charts/datadog/templates/cluster-agent-deployment.yaml

```diff
@@ -177,9 +177,13 @@ spec:
                 key: token
           - name: DD_KUBE_RESOURCES_NAMESPACE
             value: {{ .Release.Namespace }}
-          {{- if .Values.datadog.orchestratorExplorer.enabled }}
+          {{- if (.Values.datadog.orchestratorExplorer.enabled | quote) }}
```

Comment:
Why changing this check with a quote? Quoting for `value` is useful but in `{{ if }}` I don't think so.